packages/account/src/Components/leave-confirm/leave-confirm.tsx

```diff
@@ -2,44 +2,39 @@ import React from 'react';
 import { RouteComponentProps, useHistory, withRouter } from 'react-router-dom';
 import { FormikConsumer } from 'formik';
 import { Button, Icon, Modal } from '@deriv/components';
-import { isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import IconMessageContent from '../icon-message-content';
+import { useDevice } from '@deriv-com/ui';
 
 type TLeaveConfirmMessage = {
     back: () => void;
     leave: () => void;
 };
-
 type TTransitionBlocker = RouteComponentProps & {
     dirty: boolean;
     onDirty?: (prop: boolean) => void;
 };
 
 const LeaveConfirmMessage = ({ back, leave }: TLeaveConfirmMessage) => {
+    const { isMobile } = useDevice();
+
     return (
         <IconMessageContent
             className='leave-confirm'
             message={localize('Unsaved changes')}
             text={localize('You have unsaved changes. Are you sure you want to discard changes and leave this page?')}
-            icon={<Icon icon='IcUnsavedChanges' size={isMobile() ? 93 : 128} data_testid='dt_unsaved_changes_icon' />}
+            icon={<Icon icon='IcUnsavedChanges' size={isMobile ? 93 : 128} data_testid='dt_unsaved_changes_icon' />}
+            is_disabled_for_mobile={true}
         >
             <div className='account-management-flex-wrapper account-management-leave-confirm'>
-                <Button
-                    type='button'
-                    has_effect
-                    onClick={back}
-                    text={localize('Cancel')}
-                    secondary
-                    {...(isMobile() ? { large: true } : {})}
-                />
+                <Button type='button' has_effect onClick={back} text={localize('Cancel')} secondary large={isMobile} />
                 <Button
                     type='button'
                     has_effect
                     onClick={leave}
                     text={localize('Leave Settings')}
                     primary
-                    {...(isMobile() ? { large: true } : {})}
+                    large={isMobile}
```

Comment:
should it be 'isMobile' or '!isDesktop' ?

Reply:
Hi @yauheni-deriv 👋 
Thanks for pointing this out. 

It should be `isMobile` because tablet view and desktop view will be same for this. Only mobile view is different.